ios_mcp/source/gfx.h

```diff
@@ -35,9 +36,36 @@ uint32_t gfx_get_text_width(const char* string);
 typedef enum GfxPrintFlags {
     GfxPrintFlag_AlignRight     = (1U << 0),
     GfxPrintFlag_ClearBG        = (1U << 1),
+    GfxPrintFlag_Underline      = (1U << 2),
+    GfxPrintFlag_NewlinePlus4   = (1U << 3),    // Newlines should have +4
```

Comment:
Why does this need to be an additional flag?

Reply:
Underline does an automatic underline.
NewlinePlus4 adds CHAR_SIZE_DRC_Y+4 for each newline instead of just CHAR_SIZE_DRC_Y.